docs/ABOUT.md

```diff
@@ -1 +1,29 @@
-From web applications to games, the Java language is as diverse as it is portable meaning that it is found nearly everywhere. Some notable examples include Minecraft, Android apps, and large parts of websites like Amazon. With a very strong focus on object oriented programming, it is an ideal language in which to implement complex data structures and algorithms. 
+
+[Java](https://en.wikipedia.org/wiki/Java_(programming_language)) is a general-purpose computer programming language that is 
+- concurrent, 
+- [class-based](https://en.wikipedia.org/wiki/Class-based_programming), 
+- [object-oriented](https://en.wikipedia.org/wiki/Object-oriented_programming), 
+
+and specifically designed to have as few [implementation dependencies](http://www.java-tips.org/other-api-tips-100035/60-netbeans/1025-what-is-an-implementation-dependency-and-whathowwhen-should-i-use-one.html) as possible. Java applications are typically compiled to [bytecode](https://en.wikipedia.org/wiki/Bytecode) that can run on any [Java virtual machine](https://en.wikipedia.org/wiki/Java_virtual_machine) (JVM) regardless of computer architecture.
+
+There were five primary goals in the creation of the Java language:
+
+ - It must be simple, object-oriented, and familiar.
+ - It must be robust and secure.
+- It must be architecture-neutral and portable.
+- It must execute with high performance.
+- It must be interpreted, threaded, and dynamic.
+
+The language derives much of its syntax from C and C++, but it has fewer low-level facilities than either of them.
+
+Why Learn JAVA?
+
+- JAVA IS EVERYWHERE. It is one of the most in-demand programming languages, a standard for enterprise software, web-based content, games and mobile apps, as well as the Android operating system. 
+
+- Java is Platform Independent: This because of its [Write Once, Run Anywhere](https://en.wikipedia.org/wiki/Write_once,_run_anywhere)  policy.This is still one of the reasons for Java being a relatively better programming language. Most of the Java applications are developed in Windows environment and run on UNIX platform.
```

Comment:
There should be a space after the period following the word 'policy'.  The second sentence is unclear.  Better than what?  For what?

> This is still one of the reasons for Java being a relatively better programming language. 

The final sentence may be true (I have no idea-- do you have a citation?), but I'm not sure how it helps the reader.  Are you trying to support your point about it being cross-platform?

>Most of the Java applications are developed in Windows environment and run on UNIX platform.